nim_status/lib/shim.nim

```diff
@@ -8,3 +8,5 @@ proc hashMessage*(message: string): string =
 export generateAlias
 
 export identicon
+
+export createAccount
```

Comment:
I didn't notice this previously or it's new. So, the thing is that procs exported in the shim are supposed to be 100% 1-to-1 with the input/output of the same function in status-go. It's perfectly fine if that's not the case for the procs in lib, but the shim proc would need to account for that.  If we don't want to take time now to implement the shim proc, that's okay, but the lib proc shouldn't simply be re-exported as is, so for now I think it should be removed. Does that make sense?  If not, please let me know because I may be doing a bad job of explaining the library's overall design.